model/nic.rb

```diff
@@ -24,4 +24,10 @@ def self.ubid_to_name(ubid)
   def ubid_to_tap_name
     ubid.to_s[0..9]
   end
+
+  def add_firewall_rules
+    rules = private_subnet.firewall_rules
+    rule_cmds = "sudo ip netns exec #{vm.inhost_name} bash -c 'sudo nft add element inet fw_table allowed_ipv4_ips { #{rules.map { _1.start_ip4.to_s }.join(",")} }'"
```

Comment:
this is here just so that I show myself that we can add rules atomically. Probably, this will require to cleanup the ips first.